src/engraving/libmscore/score.cpp

```diff
@@ -5375,6 +5375,17 @@ Part* Score::partById(const ID& partId) const
     return nullptr;
 }
 
+int Score::visibleParts() const
```

Comment:
It should be renamed to `visiblePartCount`

Reply:
👍